src/main/resources/db/migration/V2024_10_25_1230__CIV-15300_case_proceed_caseman_task_list_update.sql

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Will this have any knock on effects to accounts in production in this state ? Will all their task states just disappear for example ?